assets/package.json

```diff
@@ -9,16 +9,18 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@openfn/compiler": "0.0.1",
+    "@monaco-editor/react": "^4.4.5",
     "@openfn/workflow-diagram": "^0.0.5",
     "blueimp-md5": "^2.19.0",
+    "monaco-editor": "^0.34.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-markdown": "^8.0.3",
     "zustand": "^4.0.0-rc.2"
   },
   "devDependencies": {
     "@types/react": "^18.0.15",
-    "@types/react-dom": "^18.0.6"
+    "@types/react-dom": "^18.0.6",
+    "@openfn/compiler": "file:../../kit/packages/compiler"
```

Comment:
Whoops - I didn't mean to check this in! I'll need to reverse it out and maybe update the compiler version number.

It's needed because I've had to expose extra exports in the compiler package.